pkgs/tools/security/pinentry/default.nix

```diff
@@ -23,6 +23,9 @@ stdenv.mkDerivation rec {
     substituteInPlace pinentry/pinentry-curses.c --replace ncursesw ncurses
   '';
 
+  patches = [ ./0006-gtk2-Fix-a-problem-with-fvwm.patch
+              ./0007-gtk2-When-X11-input-grabbing-fails-try-again-over-0..patch ];
```

Comment:
Please use `fetchpatch` to download the patches from https://anonscm.debian.org/cgit/pkg-gnupg/pinentry.git/tree/debian/patches?h=debian/1.0.0-2.